src/txn/txn_recover.c

```diff
@@ -452,6 +452,69 @@ __recovery_set_oldest_timestamp(WT_RECOVERY *r)
     return (0);
 }
 
+/*
+ * __recovery_set_checkpoint_snapshot --
+ *     Set the oldest timestamp as retrieved from the metadata file.
```

Comment:
The function description mismatches with the function what actually doing

Reply:
Fixed